### ISSUE TITLE
How to upgrade

### ISSUE THREAD

**Issue by cesardmoro** [acquia/lightning-project#83]
There is any documentatión on how to upgrade ? im looking but cannot find. 2.1.8 running drupal 8.3.9 i need to update to 8.6

**Comment by scvinodkumar** [comment]
@cesardmoro I am updating Acquia Lightning version from 3.1.5 to 4.1.1. This is first time I am doing update in Drupal8.
Could you please check and confirm is below steps is right or wrong? I am updating from 3.1.5 to 3.1.6.
- I created new branch from current master branch in the environment,.
- running below commands one by one
  - composer update
  - composer require acquia/lightning:~3.1.6 --no-update
  - composer update acquia/lightning --with-all-dependencies
  - drush updb
  - drush cr

**Comment by cesardmoro** [comment]
Yes i would recomend to run composer update before every drupal update & lightning update.

**Comment by scvinodkumar** [comment]
okay so it would be like below

composer self-update
composer update
composer require acquia/lightning:~3.1.6 --no-update
composer update acquia/lightning --with-all-dependencies
drush updb
drush cr
